Remove legacy min-width and min-height from icon mixins (T401458)

kavaljeetsingh-dev opened the pull request:
This removes the min-width and min-height declarations from the icon size mixin in codex.

Bug: T401458

egardner reviewed (changes requested):
This needs to be re-worked to get rid of the Yarn stuff.

I didn't know about the "recommendations" thing though – perhaps we could use that to recommend Volar and similar useful tools for Vue development.

catrope commented:
I've made major updates to this PR. The VSCode recommendations stuff is gone, we don't need that anymore now that we've moved away from Yarn. I've also changed it from a local eslint setup in just the vue-components package to a global setup for the entire repo, and updated existing code to fix eslint errors/warnings.

kavaljeetsingh-dev committed:
Remove legacy min-width and min-height from icon mixins (T401458)

This removes the min-width and min-height declarations from the icon size mixin in codex.

Bug: T401458
